cluster/cluster.go

```diff
@@ -298,6 +309,15 @@ func (p *Peer) register(reg prometheus.Registerer) {
 		Help: "A counter of the number of cluster peer reconnections.",
 	})
 
+	p.failedRefreshCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "alertmanager_cluster_refresh_failed_total",
+		Help: "A counter of the number of failed cluster peer refresh attempts.",
+	})
+	p.refreshCounter = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "alertmanager_cluster_refresh_total",
```

Comment:
Metric name and metric description seem diverged to me. `alertmanager_cluster_refresh_total` sounds like the amount of times a refresh happened and not amount of times a peer joined the cluster due to a refresh. How about `alertmanager_cluster_refresh_join_total`? (Not quite perfect either)

Reply:
I agree, renamed to `alertmanager_cluster_refresh_join_total` and `alertmanager_cluster_refresh_join_failed_total` to indicate that there are counters for join